cherrypy/process/servers.py

```diff
@@ -237,8 +243,22 @@ def wait(self):
             # Wait for port to be occupied if not running via socket-activation
             # (for socket-activation the port will be managed by systemd )
             if isinstance(self.bind_addr, tuple):
-                host, port = self.bind_addr
-                wait_for_occupied_port(host, port)
+                host, port = self.bound_addr
```

Comment:
I personally think `port` is clearer. When I follow the codacy link, it says "Having unused variable in a project can increase the maintainance [sic] cost, proliferating dead code and increasing the cognitive burden of the programmer." In this regard, I believe Codacy is mistaken. The variable provides clarity as to what the other part of the pair is, even if it's unused.

I do wonder, though, if it was a mistake to have removed the 'wait_for_occupied_port'.